docs/manuals/source/Configuration/Director.rst

```diff
@@ -715,32 +715,43 @@ The directives within an Options resource may be one of the following:
    It is strongly recommend to use signatures for your backups.
    Note, only one type of signature can be computed per file.
 
+   You have to find the right balance between speed and security.
+   For example the MD5 message-digest algorithm is a cryptographically broken but still
```

Comment:
```suggestion
   For example the MD5 message-digest algorithm is a cryptographically broken but, it is still suitable for other non-cryptographic purposes (like calculating a checksum to avoid unintended data change as used by Bareos here) and may be preferred due to lower computational requirements than more recent SHA algorithms.
```